Cargo.toml

```diff
@@ -7,12 +7,15 @@ license = "Apache-2.0/MIT"
 repository = "https://github.com/nikomatsakis/rayon"
 documentation = "https://docs.rs/rayon/"
 
+[workspace]
+members = ["rayon-demo", "rayon-core"]
+
 [dependencies]
+rayon-core = { path = "rayon-core" }
```

Comment:
Note, you'll have to version this before it can be published.  And this is what we'll have to raise any time there's a new `rayon-core` interface that `rayon` want to use/reexport.